pkg/executioncluster/impl/random_cluster_selector.go

```diff
@@ -130,7 +130,6 @@ func (s RandomClusterSelector) GetTarget(ctx context.Context, spec *executionclu
 		if flyteAdminError, ok := err.(errors.FlyteAdminError); !ok || flyteAdminError.Code() != codes.NotFound {
 			return nil, err
 		}
-		return nil, err
```

Comment:
I'm not comfortable with ignoring any error here... Can we be specific about the error that maps to "not found" or whatever... something along the lines of:
```
if err != nil {
		if flyteAdminError, ok := err.(errors.FlyteAdminError); !ok || flyteAdminError.Code() != codes.NotFound {
			return nil, err
		}

                if err != ErrNoOverride {
                    return nil, err
                }
}
```

Reply:
@EngHabu We are only ignoring not found error here. Line 133 is a bug

Reply:
I'm resolving this comment.